kr/components/grids_templates/column_resizing.md

```diff
@@ -144,6 +191,38 @@ public onResize(event) {
 > [!NOTE]
 > [`minWidth`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#minwidth)가 그보다 작은 값으로 설정된 경우에도 열 크기를 **88px** 미만으로 조정할 수 없습니다.
 
+Mixing the minimum and maximum column width value types (pixels or percentages) is allowed. If the values set for minimum and maximum are set to percentages, the respective column size will be limited to those exact sizes similar to pixels.
+
+This means the following configurations are possible:
+
+@@if (igxName !== 'IgxHierarchicalGrid') {
```

Comment:
I don't think  these configurations are limited to igxGrid and igxTreeGrid, not to mention how the topic would look for HierarchicalGrid with the prelude to a code sample but without one.

I think these apply to all grids and the if clauses should be removed